### DISCUSSION TITLE
How to gracefully open/refresh/close a tab according to a schedule?

### DISCUSSION THREAD

**Discussion by 1c383r9** [jlesage/docker-firefox#205] [Q&A]
Hi guys!

I tried some "tab scheduler" extensions from the store but they don't really work. It seems that the dockerized FF got hybernated while idling so the schedule extension went out of order.

Any suggestions if I want to do some scheduled tab activities? Haven't found a good solution.

Kudos!

**Comment by Okkbro501** [comment]
Use Tampermonkey script for that purpose
